tensorflow_datasets/core/download/kaggle.py

```diff
@@ -151,3 +151,83 @@ def _run_kaggle_command(command_args, competition_name):
 def _log_command_output(output, error=False):
   log = logging.error if error else logging.info
   log("kaggle command output:\n%s", tf.compat.as_text(output))
+
+
+class KaggleSearch(object):
```

Comment:
Could you move all code which is to be used from jupiter notebook to a different module please?

Reply:
I thought that but I couldn't decide. Thank you! I will move to the new file